### DISCUSSION TITLE
Repos with frequent releases are too noisy on the home feed

### DISCUSSION THREAD

**Discussion by brianpeiris** [community/community#68803] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

[Web Platform Tests (web-platform-tests/wpt)](https://github.com/web-platform-tests/wpt) is a good example of a repo that publishes new releases multiple times a day, so it dominates the home feed. I don't have watching enabled on the repo, and I'd rather not unstar the repo to fix the home feed noise.

**Comment by brianpeiris** [comment]
I also wanted to mention that I've been using the "Show less activity like this" menu for these cards for weeks and that seems to do nothing. I would have hoped it would feed into the algorithm and remove these release updates from my feed over time.

Here's an example of what the feed looks like with these noisy updates. The size of the wpt release cards makes them especially annoying.

![image](https://github.com/community/community/assets/79419/b418a9e4-54f2-473e-b0dd-092323ee4fc8)

**Comment by BigRedTower** [comment]
uzDoom turned my feed into a uzDoom feed. How many releases per-day does a DOOM game really need? Had to unstar, to see my actual feed.
